crates/oxc_transformer_plugins/tests/integrations/snapshots/integrations__replace_global_defines__test_sourcemap.snap

```diff
@@ -4,17 +4,14 @@ expression: snapshot
 ---
 - test.js.map
 (0:0) "1;\n" --> (0:0) "1;\n"
-(1:0) "__OBJECT__;\n" --> (1:0) "({ 'hello': 'test' });\n"
```

Comment:
This doesn't look right. I assume there should be a mapping for this? Or is the fault in `ReplaceGlobalDefines`?